docs/workflow/building/coreclr/nativeaot.md

```diff
@@ -50,9 +50,16 @@ For the runtime libraries:
 - Run `build clr.nativeaotlibs+libs -c [Debug|Release] -a wasm -os Browser`. This will create the architecture-dependent libraries needed for linking and runtime execution, as well as the managed binaries to be used as input to ILC. Add the `clr.nativeaotbuild` subset if you want the packages for publishing, e.g. `build clr.nativeaotlibs+clr.nativeaotbuild+libs -c Debug -a wasm -os Browser`
 
 For the compilers:
-- Download the LLVM 11.0.0 source from https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/llvm-11.0.0.src.tar.xz
-- Extract it and create a subdirectory in the `llvm-11.0.0.src` folder (`path-to-the-build-directory`).
-- Configure the LLVM source to use the same runtime as the Jit: `cmake -G "Visual Studio 17 2022" -DCMAKE_BUILD_TYPE=Debug -D LLVM_USE_CRT_DEBUG=MTd path-to-the-build-directory` or if building for the Release configuration `cmake -G "Visual Studio 17 2022" -DCMAKE_BUILD_TYPE=Release -D LLVM_USE_CRT_RELEASE=MT path-to-the-build-directory`
+- Download the LLVM 15.0.6 source from https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.6/lldb-15.0.6.src.tar.xz
+- Extract it and create a subdirectory in the `llvm-15.0.6.src` folder (`path-to-the-build-directory`).  Do not create at the root of a drive.
+- Download the CMake 15.0.6 source from https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.6/cmake-15.0.6.src.tar.xz
```

Comment:
Hmm, custom CMake is quite an unusual requirement. Why is it needed?

Reply:
I hit this problem: https://github.com/llvm/llvm-project/issues/57573#issuecomment-1237955018

Reply:
Sad. I suppose there is nothing we can do then.

Reply:
but I've not fixed then yml scripts...  better do that.